server/server.go

```diff
@@ -723,15 +724,17 @@ func killConn(conn *clientConn) {
 	// we need to CANCEL the matching jobID first.
 	if sessVars.StmtCtx.IsDDLJobInQueue {
 		jobID := sessVars.StmtCtx.DDLJobID
-		err := kv.RunInNewTxn(context.Background(), conn.ctx.GetStore(), true, func(ctx context.Context, txn kv.Transaction) error {
-			// errs is the error per job, there is only one submitted
-			// err is the error of the overall task
-			errs, err := ddl.CancelJobs(txn, []int64{jobID})
+		se, err := session.CreateSession(conn.ctx.GetStore())
```

Comment:
Can we use `conn.getCtx().Session` instead?

Reply:
better not. cancel job will create a new txn it may influence the user session

Reply:
I think it is fine because kill connection is not a frequent operation